### ISSUE TITLE
ReferenceError: explain is not defined

### ISSUE THREAD

**Issue by christianbundy** [ssbc/ssb-client#38]
New in 4.7.0?

```
ReferenceError: explain is not defined
    at /home/christianbundy/src/redacted/node_modules/ssb-client/client.js:69:13
    at /home/christianbundy/src/redacted/node_modules/multiserver/compose.js:80:24
    at onConnect (/home/christianbundy/src/redactednode_modules/multiserver/plugins/ws.js:93:11)
    at WebSocket.<anonymous> (/home/christianbundy/src/redactednode_modules/pull-ws/source.js:49:13)
    at WebSocket.onError (/home/christianbundy/src/redacted/node_modules/ws/lib/WebSocket.js:452:14)
    at WebSocket.emit (events.js:197:13)
    at ClientRequest.onerror (/home/christianbundy/src/redacted/node_modules/ws/lib/WebSocket.js:711:10)
    at ClientRequest.emit (events.js:197:13)
    at Socket.socketErrorListener (_http_client.js:397:9)
    at Socket.emit (events.js:197:13)
```

**Comment by dominictarr** [comment]
fixed in 4.7.1
looks like what happened, is that I refactored out client.js but forgot to also define explain there.

**Comment by dominictarr** [comment]
also available in ssb-server@14.1.4
